search-service/src/main/kotlin/com/egm/stellio/search/entity/service/EntityAttributeService.kt

```diff
@@ -220,7 +220,7 @@ class EntityAttributeService(
 
         // if the temporal property existed before, the create operation returned a different id than the one
         // in the attribute object
-        val timeProperty =
+        val timeProperty = // todo what is the point of choosing this?
```

Comment:
for the audit trace added below, we need to know if it was a creation or a modification (the `create(Attribute)` is actually an upsert)